feature_layers/add-feature-clustering/README.md

```diff
@@ -0,0 +1,42 @@
+# Add feature clustering
+
+Add clustering feature reduction to a point feature layer.
+
+![Image of display map](AddFeatureClustering.png)
+
+## Use case
+
+Feature clustering can be used to dynamically aggregate groups of points that are within proximity of each other in order to represent each group with a single symbol. Such grouping allows you to see patterns in the data that are difficult to visualize when a layer contains hundreds or thousands of points that overlap and cover each other. Users can add feature clustering to point feature layers. This is useful when the layer does not have the feature reduction defined or when the existing feature reduction properties need to be overridden.
+
+## How to use the sample
+
+Interact with the controls to customize clustering feature reduction properties. Click on any clustered aggregate geoelement to see the cluster feature count and aggregate fields in the popup.
+
+## How it works
+
+1. Create a map from a web map `PortalItem`.
+2. Create a `ClassBreaksRenderer` and define a `FieldName` and `DefaultSymbol`. `FieldName` must be one of the summary fields in the `AggregateFields` collection.
+3. Add `ClassBreak` objects each with an associated `SimpleMarkerSymbol` to the renderer.
+4. Create a `ClusteringFeatureReduction` using the renderer.
+5. Add `AggregateField` objects to the feature reduction where the `FieldName` is the name of the field to aggregate and the `StatisticType` is the type of aggregation to perform.
+6. Define the `MinSymbolSize` and `MaxSymbolSize` for the feature reduction. If these are not defined they default to 12 and 70 respectively.
+7. Add the `ClusteringFeatureReduction` to the `FeatureLayer`.
+8. Create a `LabelDefinition` with a `SimpleLabelExpression` and `TextSymbol` to define the cluster label.
+
+## Relevant API
+
+* AggregateGeoElement
+* ClassBreaksRenderer
+* FeatureLayer
+* FeatureReduction
+* GeoElement
+* IdentifyLayerResult
+* PopupViewer
```

Comment:
Why is this relevant API?

Reply:
I accidentally left it in, thank you for noticing - removed it now